cloudwatch_metrics_collector/src/cloudwatch_metrics_collector.cpp

```diff
@@ -35,8 +35,10 @@
 #include <string>
 #include <vector>
 
-using namespace Aws::CloudWatch::Metrics;
+using namespace Aws::Client;
```

Comment:
Keep the Aws::Client. This class should be surrounded in the following instead of "using namespace Aws::Cloudwatch::Metrics"
```
namespace Aws
{ 
namespace Cloudwatch
{
namespace Metrics
{
...
}
}
}
```